.werft/workspace-run-integration-tests.sh

```diff
@@ -84,7 +84,7 @@ werft log phase "build preview environment" "build preview environment"
     git checkout -B "${BRANCH}" && \
     git commit -m "${TEMP_COMMIT_MSG}" --allow-empty  && \
     git push --set-upstream origin "${BRANCH}" && \
-    werft run github -a with-preview=true
+    werft run github -a with-preview=true -a with-large-vm=true
```

Comment:
Cool, I don't know we have this annotation until now.

Reply:
I learned it from @vulkoingim